src/Sylius/Behat/Context/Api/Admin/ManagingOrdersContext.php

```diff
@@ -267,11 +267,15 @@ public function iMarkThisOrderAsAPaid(OrderInterface $order): void
      */
     public function iShipThisOrder(OrderInterface $order): void
     {
+        $shipment = $order->getShipments()->first();
```

Comment:
Same approach as with `$order->getLastPayment`

```suggestion
        $shipment = $order->getShipments()->last();
```

Reply:
Do you want to assert, that shipment is present?

```php
Assert::notFalse($shipment);
```